js/src/framework.ts

```diff
@@ -245,6 +245,18 @@ export interface Evaluator<
    * Optionally explicitly specify the git metadata for this experiment. This takes precedence over `gitMetadataSettings` if specified.
    */
   repoInfo?: RepoInfo;
+
+  /**
+   * Optionally supply a custom function to specifically handle score values when tasks or scoring functions have errored.
+   * If set to true, log a 0 score to the root span for any scorer that was not run. For a task that errored, all scores will be logged to the root span with a 0 value.
+   */
+  errorScoreHandler?:
```

Comment:
should we instead export defaultErrorScoreHandler and let the user set that if they want the default behavior rather than use a union with true?